Tests/TuistAutomationTests/ProjectMappers/PruneUITestsProjectMapperTests.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistSupport
+import XCTest
+
+@testable import TuistAutomation
+@testable import TuistCoreTesting
+@testable import TuistSupportTesting
+
+final class PruneUITestsProjectMapperTests: TuistUnitTestCase {
+    private var subject: PruneUITestsProjectMapper!
+
+    override func setUp() {
+        super.setUp()
+        subject = .init()
+    }
+
+    override func tearDown() {
+        subject = nil
+        super.tearDown()
+    }
+
+    func test_prune_is_set_to_ui_targets() throws {
+        // Given
+        let project = Project.test(
+            targets: [
+                .test(name: "App", product: .app),
+                .test(name: "UITests", product: .uiTests),
```

Comment:
Could we add another target here that is just `unitTest` so we can ensure we only filter uiTests

Reply:
Sure, will do 👍